operations/validate.go

```diff
@@ -151,28 +160,34 @@ func validateFile(path string, ac *legacyClient, quiet, includeLong bool, localM
 
 // loadProjectIntoWithValidation returns a warning (instead of an error) if there's an error with unmarshalling strictly
 func loadProjectIntoWithValidation(ctx context.Context, data []byte, opts *model.GetProjectOpts,
-	project *model.Project) (*model.ParserProject, validator.ValidationErrors) {
+	project *model.Project) (*model.ParserProject, *model.ProjectConfig, validator.ValidationErrors) {
 	errs := validator.ValidationErrors{}
-	pp, _, err := model.LoadProjectInto(ctx, data, opts, "", project)
-
+	pc, err := model.CreateProjectConfig(data, "")
```

Comment:
Can we add a comment here that we validate this regardless of if version control is actually being used to verify correctness if it _is_ turned on